fleetspeak/src/client/watchdog/watchdog.go

```diff
@@ -85,7 +86,7 @@ func (w *Watchdog) watch() {
 				}
 				log.Infof("Wrote goroutine traces to %s", f.Name())
 			}
-			if !w.skipExit {
+			if !w.exit {
```

Comment:
Don't you need to invert the condition here?

Reply:
done